masterlist.yaml

```
@@ -4736,6 +4736,21 @@ plugins:
       - crc: 0x3AD82805
         util: 'TES4Edit v4.0.3i (RC1 for 4.0.4)'
 
+  - name: 'ImmersivePorters.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/48704/'
+        name: 'Immersive Porters'
+    msg:
+      - <<: *wildEdits
+        subs: [ 'Re-add the **Repair** service in **000947AA** in **ImmersivePorters.esp**.' ]
```

It'd be helpful to point out where the services are located in the record:

```suggestion
        subs: [ 'Re-add the **Repair** service in the **AI Data** of **000947AA** in **ImmersivePorters.esp**.' ]
```